Userland/Applications/PixelPaint/MainWidget.cpp

```diff
@@ -264,6 +264,14 @@ ErrorOr<void> MainWidget::initialize_menubar(GUI::Window& window)
 
     TRY(file_menu->try_add_separator());
 
+    TRY(file_menu->add_recent_files_list([&](auto& action) {
+        auto path = action.text();
+        auto response = FileSystemAccessClient::Client::the().request_file_read_only_approved(&window, path);
+        if (response.is_error())
+            return;
```

Comment:
A bit surprising this is just getting dropped. I suppose it matches other uses of this (which is also weird).